easybuild/tools/repository/svnrepo.py

```diff
@@ -170,3 +170,19 @@ def cleanup(self):
             rmtree2(self.wc)
         except OSError, err:
             self.log.exception("Can't remove working copy %s: %s" % (self.wc, err))
+
+    def export(self, to_path, revision=None):
```

Comment:
I'm not sure, export doesn't really have an equivalent in other repository types (as in, a snapshot of a single revision)